config/socket/game.js

```diff
@@ -337,8 +331,10 @@ Game.prototype.persistGame = function () {
         round: this.round,
       }
     };
-    const newGame = new GameModel(payload);
-    newGame.save();
+    if (!(payload.meta.state === 'game dissolved' && payload.meta.gameWinner)) {
```

Comment:
I see you're only saving the game if it has a game winner. I thought we were saving the game regardless if the game  had a winner or it was abandoned

Reply:
Yes, we are saving a game if it was abandoned/terminated or if there is a winner.  This block of code "if (!(payload.meta.state === 'game dissolved' && payload.meta.gameWinner)) " prevents the saving of the game twice. Normally if you terminate a game it saves the game with no winner and a game state "game dissolved".  We discovered that when a game end and the game-winner is saved, when the user refreshes the browser, it saves the same game again with a game state "game dissolved" and a game-winner. You cant have a game-winner with a game state "game dissolved".